resources/metadata/gradleExecuteBuild.yaml

```diff
@@ -16,23 +20,23 @@ spec:
           - STAGES
           - STEPS
         mandatory: false
-      - name: task
-        type: string
-        description: Gradle task that should be executed.
+      - name: tasks
```

Comment:
I guess we need to keep compatibility and leave the `task` parameter in while deprecating it.